Origin: https://github.com/RPi-Distro/firmware-nonfree/tree/master/brcm
Forwarded: not-needd
Last-Update: 2018-11-28
Reviewed-by: Gunnar Wolf <gwolf@debian.org>
Description: Add a Raspberry-specific firmware file not included upstream
 The files brcm80211/brcmfmac43430-sdio.txt (for the RPi3) and
 brcm80211/brcmfmac43455-sdio.txt (for the RPi3+) are not included in
 the downloaded tarball and are required for the WiFi to be active;
 they are accompanied with its relevant licensing terms.

Index: raspi3-firmware/brcm80211/LICENSE
===================================================================
--- /dev/null
+++ raspi3-firmware/brcm80211/LICENSE
@@ -0,0 +1,205 @@
+SOFTWARE LICENSE AGREEMENT
+
+Unless you and Broadcom Corporation (“Broadcom”) execute a separate written
+software license agreement governing use of the accompanying software, this
+software is licensed to you under the terms of this Software License Agreement
+(“Agreement”).
+
+ANY USE, REPRODUCTION OR DISTRIBUTION OF THE SOFTWARE CONSTITUTES YOUR
+ACCEPTANCE OF THIS AGREEMENT.
+
+1.	DEFINITIONS.
+
+1.1.	“Broadcom Product” means any of the proprietary integrated circuit
+product(s) sold by Broadcom with which the Software was designed to be used, or
+their successors.
+
+1.2.	“Licensee” means you or if you are accepting on behalf of an entity
+then the entity and its affiliates exercising rights under, and complying with
+all of the terms of this Agreement.
+
+1.3.	“Software” shall mean that software made available by Broadcom to
+Licensee in binary code form with this Agreement.
+
+2.	LICENSE GRANT; OWNERSHIP
+
+2.1.	License Grants.  Subject to the terms and conditions of this Agreement,
+Broadcom hereby grants to Licensee a non-exclusive, non-transferable,
+royalty-free license (i) to use and integrate the Software in conjunction with
+any other software; and (ii) to reproduce and distribute the Software complete,
+unmodified and as provided by Broadcom, and only for use with a Broadcom
+Product.
+
+2.2.	Restriction on Modification.  Licensee may not make any modifications
+to the Software.
+
+2.3.	Restriction on Distribution.  Licensee shall only distribute the
+Software under the terms of this Agreement and a copy of this Agreement
+accompanies such distribution.
+
+2.4.	Proprietary Notices.  Licensee shall not remove, efface or obscure any
+copyright or trademark notices from the Software.  Licensee shall include
+reproductions of the Broadcom copyright notice with each copy of the Software,
+except where such Software is embedded in a manner not readily accessible to
+the end user.  Licensee acknowledges that any symbols, trademarks, tradenames,
+and service marks adopted by Broadcom to identify the Software belong to
+Broadcom and that Licensee shall have no rights therein.
+
+2.5.	Ownership.  Broadcom shall retain all right, title and interest,
+including all intellectual property rights, in and to the Software.  Licensee
+hereby covenants that it will not assert any claim that the Software created by
+or for Broadcom infringe any intellectual property right owned or controlled by
+Licensee; provided however, the foregoing shall not apply in case the Agreement
+is terminated.
+
+2.6.	No Other Rights Granted; Restrictions.  Apart from the license rights
+expressly set forth in this Agreement, Broadcom does not grant and Licensee
+does not receive any ownership right, title or interest nor any security
+interest or other interest in any intellectual property rights relating to the
+Software, nor in any copy of any part of the foregoing.  No license is granted
+to Licensee in any human readable code of the Software (source code). Licensee
+shall not (i) use, license, sell or otherwise distribute the Software except as
+provided in this Agreement, (ii) attempt to modify in any way, reverse
+engineer, decompile or disassemble any portion of the Software; or (iii) use
+the Software or other material in violation of any applicable law or
+regulation, including but not limited to any regulatory agency, such as FCC,
+rules.
+
+3.	NO WARRANTY OR SUPPORT
+
+3.1.	No Warranty. THE SOFTWARE IS OFFERED “AS IS,” AND BROADCOM GRANTS AND
+LICENSEE RECEIVES NO WARRANTIES OF ANY KIND, EXPRESS OR IMPLIED, BY STATUTE,
+COMMUNICATION OR CONDUCT WITH LICENSEE, OR OTHERWISE.  BROADCOM SPECIFICALLY
+DISCLAIMS ANY IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A SPECIFIC
+PURPOSE OR NONINFRINGEMENT CONCERNING THE SOFTWARE OR ANY UPGRADES TO OR
+DOCUMENTATION FOR THE SOFTWARE.  WITHOUT LIMITATION OF THE ABOVE, BROADCOM
+GRANTS NO WARRANTY THAT THE SOFTWARE IS ERROR-FREE OR WILL OPERATE WITHOUT
+INTERRUPTION, AND GRANTS NO WARRANTY REGARDING ITS USE OR THE RESULTS THEREFROM
+INCLUDING, WITHOUT LIMITATION, ITS CORRECTNESS, ACCURACY OR RELIABILITY.
+
+3.2.	No Support.  Nothing in this agreement shall obligate Broadcom to
+provide any support for the Software. Broadcom may, but shall be under no
+obligation to, correct any defects in the Software and/or provide updates to
+licensees of the Software.  Licensee shall make reasonable efforts to promptly
+report to Broadcom any defects it finds in the Software, as an aid to creating
+improved revisions of the Software.
+
+3.3.	Dangerous Applications.  The Software is not designed, intended, or
+certified for use in components of systems intended for the operation of
+weapons, weapons systems, nuclear installations, means of mass transportation,
+aviation, life-support computers or equipment (including resuscitation
+equipment and surgical implants), pollution control, hazardous substances
+management, or for any other dangerous application in which the failure of the
+Software could create a situation where personal injury or death may occur. 
+Licensee understands that use of the Software in such applications is fully at
+the risk of Licensee.
+
+4.	TERM AND TERMINATION
+
+4.1.	Termination.  This Agreement will automatically terminate if Licensee
+fails to comply with any of the terms and conditions hereof. In such event,
+Licensee must destroy all copies of the Software and all of its component
+parts.
+
+4.2.	Effect Of Termination.  Upon any termination of this Agreement, the
+rights and licenses granted to Licensee under this Agreement shall immediately
+terminate.
+
+4.3.	Survival.  The rights and obligations under this Agreement which by
+their nature should survive termination will remain in effect after expiration
+or termination of this Agreement.
+
+5.	CONFIDENTIALITY
+
+5.1.	Obligations.  Licensee acknowledges and agrees that any documentation
+relating to the Software, and any other information (if such other information
+is identified as confidential or should be recognized as confidential under the
+circumstances) provided to Licensee by Broadcom hereunder (collectively,
+“Confidential Information”) constitute the confidential and proprietary
+information of Broadcom, and that Licensee’s protection thereof is an essential
+condition to Licensee’s use and possession of the Software.  Licensee shall
+retain all Confidential Information in strict confidence and not disclose it to
+any third party or use it in any way except under a written agreement with
+terms and conditions at least as protective as the terms of this Section.
+Licensee will exercise at least the same amount of diligence in preserving the
+secrecy of the Confidential Information as it uses in preserving the secrecy of
+its own most valuable confidential information, but in no event less than
+reasonable diligence.  Information shall not be considered Confidential
+Information if and to the extent that it: (i) was in the public domain at the
+time it was disclosed or has entered the public domain through no fault of
+Licensee; (ii) was known to Licensee, without restriction, at the time of
+disclosure as proven by the files of Licensee in existence at the time of
+disclosure; or (iii) becomes known to Licensee, without restriction, from a
+source other than Broadcom without breach of this Agreement by Licensee and
+otherwise not in violation of Broadcom’s rights.
+
+5.2.	Return of Confidential Information.  Notwithstanding the foregoing, all
+documents and other tangible objects containing or representing Broadcom
+Confidential Information and all copies thereof which are in the possession of
+Licensee shall be and remain the property of Broadcom, and shall be promptly
+returned to Broadcom upon written request by Broadcom or upon termination of
+this Agreement.
+
+6.	LIMITATION OF LIABILITY TO THE MAXIMUM EXTENT PERMITTED BY LAW, IN NO
+EVENT SHALL BROADCOM OR ANY OF BROADCOM’S LICENSORS HAVE ANY LIABILITY FOR ANY
+INDIRECT, INCIDENTAL, SPECIAL, OR CONSEQUENTIAL DAMAGES, HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER FOR BREACH OF CONTRACT, TORT (INCLUDING
+NEGLIGENCE) OR OTHERWISE, ARISING OUT OF THIS AGREEMENT, INCLUDING BUT NOT
+LIMITED TO LOSS OF PROFITS, EVEN IF SUCH PARTY HAS BEEN ADVISED OF THE
+POSSIBILITY OF SUCH DAMAGES. IN NO EVENT WILL BROADCOM’S LIABILITY WHETHER IN
+CONTRACT, TORT (INCLUDING NEGLIGENCE), OR OTHERWISE, EXCEED THE AMOUNT PAID BY
+LICENSEE FOR SOFTWARE UNDER THIS AGREEMENT.  THESE LIMITATIONS SHALL APPLY
+NOTWITHSTANDING ANY FAILURE OF ESSENTIAL PURPOSE OF ANY LIMITED REMEDY.
+
+7.	MISCELLANEOUS
+
+7.1.	Export Regulations. YOU UNDERSTAND AND AGREE THAT THE SOFTWARE IS
+SUBJECT TO UNITED STATES AND OTHER APPLICABLE EXPORT-RELATED LAWS AND
+REGULATIONS AND THAT YOU MAY NOT EXPORT, RE-EXPORT OR TRANSFER THE SOFTWARE OR
+ANY DIRECT PRODUCT OF THE SOFTWARE EXCEPT AS PERMITTED UNDER THOSE LAWS.
+WITHOUT LIMITING THE FOREGOING, EXPORT, RE-EXPORT OR TRANSFER OF THE SOFTWARE
+TO CUBA, IRAN, NORTH KOREA, SUDAN AND SYRIA IS PROHIBITED.
+
+7.2	Assignment. This Agreement shall be binding upon and inure to the
+benefit of the parties and their respective successors and assigns, provided,
+however that Licensee may not assign this Agreement or any rights or obligation
+hereunder, directly or indirectly, by operation of law or otherwise, without
+the prior written consent of Broadcom, and any such attempted assignment shall
+be void.  Notwithstanding the foregoing, Licensee may assign this Agreement to
+a successor to all or substantially all of its business or assets to which this
+Agreement relates that is not a competitor of Broadcom.
+
+7.3.	Governing Law; Venue.  This Agreement shall be governed by the laws of
+California without regard to any conflict-of-laws rules, and the United Nations
+Convention on Contracts for the International Sale of Goods is hereby excluded.
+The sole jurisdiction and venue for actions related to the subject matter
+hereof shall be the state and federal courts located in the County of Orange,
+California, and both parties hereby consent to such jurisdiction and venue.
+
+7.4.	Severability.  All terms and provisions of this Agreement shall, if
+possible, be construed in a manner which makes them valid, but in the event any
+term or provision of this Agreement is found by a court of competent
+jurisdiction to be illegal or unenforceable, the validity or enforceability of
+the remainder of this Agreement shall not be affected if the illegal or
+unenforceable provision does not materially affect the intent of this
+Agreement.  If the illegal or unenforceable provision materially affects the
+intent of the parties to this Agreement, this Agreement shall become
+terminated.
+
+7.5.	Equitable Relief.  Licensee hereby acknowledges that its breach of this
+Agreement would cause irreparable harm and significant injury to Broadcom that
+may be difficult to ascertain and that a remedy at law would be inadequate.
+Accordingly, Licensee agrees that Broadcom shall have the right to seek and
+obtain immediate injunctive relief to enforce obligations under the Agreement
+in addition to any other rights and remedies it may have.
+
+7.6.	Waiver.  The waiver of, or failure to enforce, any breach or default
+hereunder shall not constitute the waiver of any other or subsequent breach or
+default.
+
+7.7.	Entire Agreement.  This Agreement sets forth the entire Agreement
+between the parties and supersedes any and all prior proposals, agreements and
+representations between them, whether written or oral concerning the Software.
+This Agreement may be changed only by mutual agreement of the parties in
+writing.
+
Index: raspi3-firmware/brcm80211/brcmfmac43430-sdio.txt
===================================================================
--- /dev/null
+++ raspi3-firmware/brcm80211/brcmfmac43430-sdio.txt
@@ -0,0 +1,59 @@
+# NVRAM file for BCM943430WLSELG
+# 2.4 GHz, 20 MHz BW mode
+
+# The following parameter values are just placeholders, need to be updated.
+manfid=0x2d0
+prodid=0x0726
+vendid=0x14e4
+devid=0x43e2
+boardtype=0x0726
+boardrev=0x1202
+boardnum=22
+macaddr=00:90:4c:c5:12:38
+sromrev=11
+boardflags=0x00404201
+boardflags3=0x08000000
+xtalfreq=37400
+#xtalfreq=19200
+nocrc=1
+ag0=255
+aa2g=1
+ccode=ALL
+
+pa0itssit=0x20
+extpagain2g=0
+
+#PA parameters for 2.4GHz, measured at CHIP OUTPUT
+pa2ga0=-168,7161,-820
+AvVmid_c0=0x0,0xc8
+cckpwroffset0=5
+
+# PPR params
+maxp2ga0=84
+txpwrbckof=6
+cckbw202gpo=0
+legofdmbw202gpo=0x66111111
+mcsbw202gpo=0x77711111
+propbw202gpo=0xdd
+
+# OFDM IIR :
+ofdmdigfilttype=18
+ofdmdigfilttypebe=18
+# PAPD mode:
+papdmode=1
+papdvalidtest=1
+pacalidx2g=32
+papdepsoffset=-36
+papdendidx=61
+
+il0macaddr=00:90:4c:c5:12:38
+wl0id=0x431b
+
+deadman_to=0xffffffff
+# muxenab: 0x1 for UART enable, 0x2 for GPIOs, 0x8 for JTAG
+muxenab=0x1
+# CLDO PWM voltage settings - 0x4 - 1.1 volt
+#cldo_pwm=0x4
+
+#VCO freq 326.4MHz
+spurconfig=0x3 
Index: raspi3-firmware/brcm80211/brcmfmac43455-sdio.txt
===================================================================
--- /dev/null
+++ raspi3-firmware/brcm80211/brcmfmac43455-sdio.txt
@@ -0,0 +1,97 @@
+# Cloned from bcm94345wlpagb_p2xx.txt 
+NVRAMRev=$Rev: 498373 $
+sromrev=11
+vendid=0x14e4
+devid=0x43ab
+manfid=0x2d0
+prodid=0x06e4
+#macaddr=00:90:4c:c5:12:38
+macaddr=b8:27:eb:74:f2:6c
+nocrc=1
+boardtype=0x6e4
+boardrev=0x1304
+
+#XTAL 37.4MHz
+xtalfreq=37400
+
+btc_mode=1
+#------------------------------------------------------
+#boardflags: 5GHz eTR switch by default
+#            2.4GHz eTR switch by default
+#            bit1 for btcoex
+boardflags=0x00480201
+boardflags2=0x40800000
+boardflags3=0x48200100
+phycal_tempdelta=15
+rxchain=1
+txchain=1
+aa2g=1
+aa5g=1
+tssipos5g=1
+tssipos2g=1
+femctrl=0
+AvVmid_c0=1,165,2,100,2,100,2,100,2,100
+pa2ga0=-129,6525,-718
+pa2ga1=-149,4408,-601
+pa5ga0=-185,6836,-815,-186,6838,-815,-184,6859,-815,-184,6882,-818
+pa5ga1=-202,4285,-574,-201,4312,-578,-196,4391,-586,-201,4294,-575
+itrsw=1
+pdoffsetcckma0=2
+pdoffset2gperchan=0,-2,1,0,1,0,1,1,1,0,0,-1,-1,0
+pdoffset2g40ma0=16
+pdoffset40ma0=0x8888
+pdoffset80ma0=0x8888
+extpagain5g=2
+extpagain2g=2
+tworangetssi2g=1
+tworangetssi5g=1
+# LTECX flags
+# WCI2
+ltecxmux=0
+ltecxpadnum=0x0504
+ltecxfnsel=0x22
+ltecxgcigpio=0x32
+
+maxp2ga0=80
+ofdmlrbw202gpo=0x0022
+dot11agofdmhrbw202gpo=0x4442
+mcsbw202gpo=0x98444422
+mcsbw402gpo=0x98444422
+maxp5ga0=82,82,82,82
+mcsbw205glpo=0xb9555000
+mcsbw205gmpo=0xb9555000
+mcsbw205ghpo=0xb9555000
+mcsbw405glpo=0xb9555000
+mcsbw405gmpo=0xb9555000
+mcsbw405ghpo=0xb9555000
+mcsbw805glpo=0xb9555000
+mcsbw805gmpo=0xb9555000
+mcsbw805ghpo=0xb9555000
+
+swctrlmap_2g=0x00000000,0x00000000,0x00000000,0x010000,0x3ff
+swctrlmap_5g=0x00100010,0x00200020,0x00200020,0x010000,0x3fe
+swctrlmapext_5g=0x00000000,0x00000000,0x00000000,0x000000,0x3
+swctrlmapext_2g=0x00000000,0x00000000,0x00000000,0x000000,0x3
+
+vcodivmode=1
+deadman_to=481500000
+
+ed_thresh2g=-54
+ed_thresh5g=-54
+eu_edthresh2g=-54
+eu_edthresh5g=-54
+ldo1=4
+rawtempsense=0x1ff
+cckPwrIdxCorr=3
+cckTssiDelay=150
+ofdmTssiDelay=150
+txpwr2gAdcScale=1
+txpwr5gAdcScale=1
+dot11b_opts=0x3aa85
+cbfilttype=1
+fdsslevel_ch11=6
+
+# Improved Bluetooth coexistence parameters from Cypress
+btc_mode=1
+btc_params8=0x4e20
+btc_params1=0x7530
Index: raspi3-firmware/brcm80211/brcmfmac43455-sdio.clm_blob.base64
===================================================================
--- /dev/null
+++ raspi3-firmware/brcm80211/brcmfmac43455-sdio.clm_blob.base64
@@ -0,0 +1,247 @@
+QkxPQjwAAAD6JLl0AQAAAAIAAAAAAAAAPAAAAJQ2AAApsaz/AAAAAAAAAADQNgAABAAAAIpEUv8A
+AAAAQ0xNIERBVEEAAAwAAgA5LjEwLjEwNQAAAAAAAAAAAAAAADEuMjkuNAAAAAAAAAAAKBoAAENs
+bUltcG9ydDogMS4zNi4zAAAAAAAAAAAAAAAAAEJyb2FkY29tLTAuMAAAAAAAAAAAAABRMQBRMgBR
+MgEAAAAIAAAA4BgAAGgZAAADAwMEAwgDCQMLBAQEBwQIBAoECwUHBQsICAkJCQsLCyYmJi4mNiY+
+Jm4mdiZ+JoYmjiafJq8uLjY2Nj42bjZ2NoY+Pj5mPoZmZmZuZnZmfmaGZo5mn25ubnZufm6Gbo6G
+hoaOhpeGn46Ol5eXn5evn59BRQFBUgFBVAFBVQJCRQFCRwFCTgFDQQJDSAFDWQFDWgFERQNESwFF
+RQFFUwFGSQFGUgFHQgFHUgFIUgFIVQFJRAVJRQFJUwFJVAFKUAJLUgJLVwFMSQFMVAFMVQFMVgFN
+QQFNVAFNWAFOTAFOTwFQTAFQVAFQWQFSTwFSVQFTRQFTSQFTSwFUUgdUVwJVUwMGAgIBAQEBQUUB
+QVIBQVQBQVUCQkUBQkcBQk4BQ0ECQ0gBQ1kBQ1oBREUDREsBRUUBRVMBRkkBRlIBR0IBR1IBSFIB
+SFUBSUQFSUUBSVMBSVQBSlACS1ICS1cBTEkBTFQBTFUBTFYBTUEBTVQBTVgBTkwBTk8BUEwBUFQB
+UFkBUk8BUlUBU0UBU0kBU0sBVFcCVVMCBgICAQEBAQAGAgUBAQEHAgQBAQECCgQCAgEBARECAgEB
+AQEBAP8BHgMAA0wCBEwDAkIQBAEA/wEeAwAHSAACOgAERAcESgoCSgwERA0EQhAEAQD/AR4DAAJA
+AwI4AwQBAP8BHgMABUgAAkwCBEwJAkgQAkIQBAEA/wEeAwAEOgAERAMCQgkEOhAEAQD/AR4DAAVG
+AARQAgJQCQRMEAJIEAQBAP8BHgMABkAABEQDAkIHBEQMBEINBEAQBAEA/wEeAwAGQAAERAMCQgcE
+RAwEQg0EQBAEAQD/AR4DAARQAgJIAgROEAJKEAQBAP8BHgMAAy4CAjsDBDIQAgEA/wEeAwAFTAAC
+RgAETgkCSAoETBACAQD/AR4DAAVcAAJWAAR+CQFcEAJWEAQBAP8BHgMABUYABFACAlAJBEwQAkgQ
+BAIA/wEeBQADVAEBWgwBVA8BAgT/ARQFAgFMBQECAP8BFAUCAVYFAQIABgEUBQIBTAUCAgD/ARQF
+AAI8BQJABQQCAP8BFAUAAUwFAQIABgEUBQAFMAAERAUCOAgEMA4EOBIEAgD/ARQFAAI8BQIoBQQB
+AP8BFAMAAjwDAigDBAIA/wEUBQACTAUCSgUEAgD/ARQFAAJMBQJKBQQCAP8BFAUAAkwFAkoFBAIA
+/wEUBQACTAUCSgUEAgD/ARQFAAFIBQEBAP8BFAMAASgDAQIA/wEUBQAFMAAERAUCOAgEMA4EOBIE
+AgD/ARQFAAUwAAREBQI4CAQwDgQ4EgQDAAgBFAYCBUwCBEwFAkYQBEwSBDYVAgMACAEUBgAGPAAE
+RAMCQAoEMBICLBIEQBUCAgD/ARQFAAJMBQJKBQQCAP8BFAUAAkwFAkoFBAMACAEUBgADQAQEQAYC
+LhQEAgAGARQFAANAAwEwEgIsEgQDAAgBFAYABj4ABEIFAkIJBD4QBEISBCoVAgMA/wEUBgICVAUE
+TgYCAgD/ARQFAAFiBQECAP8BFAUAAUgFAQIA/wEUBQICTgUCVAUEAgD/AR4FAAF4BQEAAP8AAAAC
+AP8BFwUCAVwFAQIA/wEXBQIBVAUBAgD/ARcFAAJMBQJKBQQCAP8BJAUCAWQFAQIA/wEkBQIBZAUB
+AgD/ARQFAAIwBQI6BQQCBAYBFAUCAUwFAQEA/wEUAwIBTAMBAgD/ARQFBAM4AARACwQ4FAQCAUwF
+AgIA/wEUBQACUAUCPAUEAgT/ARQFAAJMBQJOBQQCAP8BFAUAAlgFAlQFBEowAEoxAEoyAEozAEo0
+AEo1AEo2AEo3AEo4AEo5AAAEAAAAwAcAAAoMDQ4PEBESExQVDAABAgMEBQYHCAkKCwQAAQIDCAwN
+Dg8QERITCAQFBgcICQoLAQEBAgEKAQsBDAENAQ4CAgIJAgoCCwIMAwkKCgoLCg0LCwwMDA0MDg0N
+Dg4iLiQkJCgkLCQwJDwkQCRkJHQkhCSMJJAkoSSlJLEoLCg8LCwwMDA0MDw0NDQ8NEA0dDR8NIw0
+kDSlODw4QEBAQGRAdGRkZHBkdGR8ZIRkiGSMZJBkpWh0aHhofGiAaIhojGildHx0jHSQeIx8fISI
+hIyEkISljIyMkIylkJCQpZWVlaGVpZWxmaGZpaWlAAEDAQUBBgEWAhYaBBYaL1cCFhwCFiABGgQa
+LU5YAxotWAQaLk5YBBouT1gDGi9XAxowWAMaMVgDGjFZAhoyAhpXAhpYARwEHDpOWAMcO1cDHDxY
+Ahw+Axw/VwIcQAIcTgMcTlcCHFACHFcCHFgDHk5YAx5PWAIeUAIfVwIfWAEgAiBYAiFYASIBIwEk
+Ay0+WAItWAM0PlgCNEACNFgBPgFXAVhFMABYQQBYVQBBVQJKUAFBRQFBVQJCTgFKUAFLUgFNWAFS
+VQEAAAADAAAAJBkAABAEBQYHCAkKCwwNDg8QERITEgQFBgcICQoLDA0ODxAREhMUFSIuBCRABGSQ
+BJWxBDIwMTgtMDMtMDkgMTg6NTY6MjgABgJUGgBUWAAHAlgRAVg2AQoCWAAAWBQABAJAGgJAWAIB
+AigRACg2AAQEOBoAOi0APD8ARlgABQM0EwFAKQFINgEIAzQBAEAMAEgUAAQEOBoAOi0APD8ARlgA
+BQM0EwFAKQFINgEIAzQBAEAMAEgUAAQDPBwCPD4CPFgCAQM4EQA8IAA8NgAEAk4cAE5YAAUCThMB
+RjYBCAJOAQBKFAAEBE4cAFQ5AFJKAFRYAAUDThMBSikBTjYBCANOAQBMDABOFAAGAVQcAAcBVBMB
+CgFUAQAEATwcAgEBOBMABgFUHAAHAVgTAQoBWAEABgJUHABUWAAHAlQTAVQ2AQoCVAEAVBQAAAAE
+AkIcAkJXAgECNhMANjYABgJcHABQWAAHAlgTAUw2AQoCWAEATBQABAJCHAJAWAIBAjgTADg2AAQB
+OBoCAQEwEQAEATwaAgEBOBEABANOHABUOQBSSQAFAk4TAUooAQgCTgEATAsABAE8GgIBATgRAAYC
+VCEAcFgABwJYGAF0NgEKAlgEAHQUAAQFOBoCRiwCOjUCRD4CRlgCAQc0EABCGwBEHAA0IQA+JABI
+LgBKNgAECzoaAFQrAEwzAEw1AEg4AFBFAERRAFBUAFBWAFBaAFBcAAUKOhABQhsBUBwBPiEBQiQB
+Ui0BSjABTDQBSDUBUDgBCAU0AAA6BwAyCgBSDwBIFAAECTwaAFArAFAzAFA1AEw9AEhRAEhUAExW
+AFBbAAUKPhABPhsBTBwBQCEBTCQBUC0BSjABSjQBRDUBUDgBCAY6AABABwBMCgBMDgBQEgBIFAAE
+BDoaAFBWAFBaAFBcAAUEOhABQhsBSDUBUDgBCAI0AABIFAAEBjgaAkQsAjg1AkQ4AkRGAkRYAgEG
+MBEARBwAMCEAPiQASi4ASDYABAg4GgBMLABENQBAOABMRQBAUQBAVABMWAAFCDYQAT4bAUYcATgh
+ATAkAUYtAUAyAUw4AQgDMAAAgAkAQBQABAU4GgBOLABANQBEOABORwAFCj4QAT4bAU4cATIhATgk
+AU4tAU4wAU40ATw1AU44AQgGOgAAQAcATAoATA4AThIASBQABAF4JAAFAXgaAQgBeAYABAF4JAAF
+AXgaAQgBeAYABAQ4GgJGLAI6NQJGWAIBBDARAD4cADAhADw2AAQEOBoCQiwCPDUCQlgCAQM4EQA8
+HQA8NgAGAlQcAFRYAAcCWBMBWDYBCgJYAQBYFAAGAlQcAGRYAAcCVBMBZDYBCgJUAQBkFAAEBDga
+AE4sAEA1AE5YAAUGPhABPhsBThwBMiEBPDUBTjgBCAM6AABABwBIFAAEBDgaAkYsAjo1AkZYAgEE
+MBEAPhwAMCEAPDYABAI8LQJEWAIBAjgdADg2AAYCcC0AcFgABwJwHQFwNgEKAnAHAHAUAAACQBYC
+QBwCBAI4GgBUVwAFAjQRAVA2AQgCMAAATBQABAJEGgBUVwAFAjQRAVA2AQgCMAAATBQABAY4GgJQ
+LAJINgJQQgJQTAJQVwIBBjYRAEAcADQhADAkAEAsAEA2AAQEMBYAMBoAPC8APFcABQIwFQEwNgEI
+AjACADAUAAQDRBoAVC8AVFcABQM0EQFQHwFQNgEIAzAAAEwIAEwUAAQFOBoBRBoCUi0AUTsASFcA
+BQM0EQFQHwFQNgEIAzAAAEwIAEwUAAQCOBoCRFgCAQIwEQA8NgAEBEAaAFAuAFBPAExYAAUENBEB
+TB4BTDEBTDYBCARAAABICABIEgBMFAAEBTgaAkAtAkQ6AkROAkRYAgEFQhEAQh0AQiUAQjAAQjYA
+BAdKGgBILABQNQBCPQBAUQA+VABOWAAFBkQRAT4dATwnAVowAT40AUY2AQgESAAAQAcAOgwAShQA
+BAkoGQAkKAAiLAAfNgAiQQAhTgAhVgAlWgAiXAAFCDgQATYbATIcAS4hASgkAScrATAwATA2AQgE
+NQAALAcALQoALRQABAooGAAnJwAkKAAiLAAfNgAiQQAhTgAiVgAjWgAkXAAFCDgQATYbATIcAS4h
+ASgkAScrATAwAS82AQgEPwAALAcALQoALRQABAY+GgBaLABQNgBaRABWTwB2WQAFBkoRAWIcAUIh
+AUokAWIvAXY3AQgFSgAATgcAUgsAYhIAdhUABAVaGwBQNgBaRABWTwB2WAAFBWISAUIhAUokAWIv
+AXY2AQgETgEAUgsAYhIAdhQABAVaGwBQNgBaQQBWTwB2WAAFBmISAUIhAUokAWIrAWIxAXY2AQgE
+TgEAUgoAYhIAdhQABAY+GgBaLABQNgBaQQBWTwB2WAAFB0oRAWIcAUIhAUokAWIrAWIxAXY2AQgF
+SgAATgcAUgoAYhIAdhQABANEHQA8RQBEUwAFBToRAUQcATwiAUItAUQzAQgEPAAAOAcAOgoARBAA
+BAREHQA8QQA8TQBEUwAFBjoRAUQcATwiAUIrAUIxAUQ2AQgEPAAAOAcAOgoARBMABgFUIgAHAVgZ
+AQoBWAUABgFUIgAHAVgZAQoBWAUABgFUIwAHAVgZAQoBWAUABAROHABUOQBSSgBOVwAFA04TAUop
+AUY2AQgDTgEATAwAShQABAI4GgA8WAAFAjARATw2AQgCQAAARhQABgRUGgJMLQJMTgJMWAIDBFAR
+AEgdAEgwAEg2AAYCVhwAVk4ABwJWEwFWMAEKAVYBAAQDThwAUk8AVFgABQNOEwFKMQFONgEIA04B
+AEwSAE4UAAYCVB8CVFgCAwJUFgBUNgAGA1QcAFg8AFhYAAcDWBMBXCcBXDYBCgNYAQBcCwBcFAAG
+AkgcAEhOAAMCTBMBTDABBAI8HAI8PgIBAjgTAEAoAAQCPBwCPD4CAQE4FwAEAlQcAlQ+AgEBOBcA
+BgFUIAAHAVgXAQoBWAMABgFUIAAHAVgXAQoBWAMABAI8HAI8PgIBAioTAEAoAAQCYhwAZj4ABQJi
+EwFmKAEIAmIBAGYLAAQERBkAQioAQDUASD4ABQJCEwFIKAEIA0AAAD4HAEgLAAQDQhsAQDUASD4A
+BQNCEgE+IQFIKAEIA0AAAD4HAEgLAAQBTCAABQNIEQFEHAFAIwEIA0QAAEAHADwLAAYCVBwAcD4A
+AwJEEwFgKAEGAlQcAHA+AAcCWBMBdCgBCgJYAQB0CwAEAkIcAEI+AAECKhMBQCgBBAI+HAJCPgIB
+AioTAEAoAAQCPBwAPD4AAQI4EwFAKAEEAigcACRXAAUCKBMBJDYBCAIkAQAkFAAEBE4cAFQ5AFJI
+AFJXAAUDThMBSiYBSjYBCANOAQBMCgBMFAAEBCAcABw6ABxOABxYAAUEIBMBHCUBHDABHDYBCAMg
+AQAcCgAcFAAECTAXADglADApADgzADA2ADhBADhNADBRACxYAAUDJBQBJDABLDYBCAIgAgAsFAAE
+CDAXADglADApADgzADA1ADhNADBRACxXAAUDJBMBJDABLDYBCAIgAQAsFAAECDAXADglADApADgz
+ADA2ADhFADBRACxYAAUCJBcBLDYBCAIgAwAsFAAEASggAAUBKBcBCAEkAwAEAiggACRYAAUCKBcB
+JDYBCAIkAwAkFAAEASgcAAUBKBMBCAEkAQAEASRXAAUBJDYBCAEkFAAEAigfACRXAAUCKBYBJDYB
+CAIkAwAkFAAEAygeAChOACRYAAUDKBQBKDABJDYBCAIkAgAkFAAEAigcACRYAAUCKBMBJDYBCAIk
+AQAkFAAEAkQgACRYAAUCSBcBJDYBCAJIAwAiFAAEAU4cAAUBThMBCAFOAQAAAAQEPBwCPDoCPE4C
+PFgCAQQ4EwA8JQA8MAA8NgAGAlQeAFRQAAcCWBQBWDMBCgJYAgBYEwAEAjwaADYwAAUCSBEBQCAB
+CANGAAA+BwBACwAGAlQcAFRYAAcCWBMBWDYBCgJYAQBYFAAGAlQcAFRYAAcCWBMBWDYBCgJYAQBY
+FAAEAjwcAjxYAgECOBMAPDYAAAI8HAI8WAIGAlQcAFRYAAMCWBMBWDYBBgFUIwAHAVQZAQoBVAUA
+BgJUIABUWAAHAlQXAVQ2AQoCVAMAVBQABgFUIwAHAVgZAQoBWAUABgJUHABwQAAHAlgTAXQqAQoC
+WAEAdA0ABAM8HAI8PgI8WAIBAzgTAEAoADw2AAQBPFgCAQE0NgAGAVBYAAcBUDYBCgFQFAAGAVBY
+AAcBUDYBCgFQFAAGAlAaAFBYAAcCUBEBUDYBCgJQAABQFAAGAUxYAAcBTDYBCgFMFAAGAUxXAAcB
+RDYBCgFEFAAEATxYAgEBPDYABgFUWAAHAVg2AQoBWBQABAFOWAAFAU42AQgBThQABAM4NAJEPgJE
+WAIBBDAhAD4kAEouAEg2AAQETjMAQDUARDgATkcABQc+IQE4JAFOLQFOMAFONAE8NQFOOAEIBEwK
+AEwOAE4SAEgUAAQDOC0COj4CPlgCAQQuHAAwIQA4KAA4NgAEAkQ0AkRYAgECQiEAPDYAAgFkVwIC
+AVRXAAQBUlcAAQFSNgESBAUGBwgJCgsMDQ4PEBESExQVBAVMAANMBwNMDANMDQNCEAMBBD4AAEIG
+AD4MADoNAAQDOgADQgkDOhADAQUqAAAyBQA8CgA2DAAuDQAEATgDAwEBIAMABAU6AANEBwNKDANE
+DQNCEAMBBTAAADoFAEIKADoMADYNAAQDQAEARAwAQA4AAQUyAAE4BQE8CgE4DAE6DQEEAkICAEAQ
+AAEBQgMBBAI5AgA2EAABAjgCATYNAQQEVAAAVAcAWgwAVA8AAQJUAQFaCwEEA0ABAEQMAEAOAAEF
+MgABOAUBPAoBOAwBOg0BBAVGAABQBwBQDABQDQBIEAABBToAAUQFAVAKAUQMATwNAQQDRgADSAkD
+SBADAQU+AABCBQBGCgBCDAA+DQAEA1YAAH4JAFYQAAEDTgABfgcBTg0BBAVGAABQBwBQDABQDQBI
+EAABBToAAUQFAVAKAUQMATwNAQQBQAUAAQFABAEEAUwFAwEBSAQAAAQwAAA4CAAwDgA4EgAAASgF
+AAABKAMAAAEoAwAABDAAADgIADAOADgSAAAEMAAAOAgAMA4AOBIAAgFWBQAAAUAFAAQEPAADQAoD
+LBEDLBQDAQMoAAA0BwAuDgAEAUoFAAEBSgQBBAFKBQABAUoEAQACQAQDLhQDBAJAAwAsEgABAzYA
+AUAIATQPAQYBTAUAAwFMBAEGAUwFAAMBgAQBBAFABQABAUIEAQQBSgUAAQFKBAEEAUoFAAEBSgQB
+BAFKBQABAUoEAQQBSgUAAQFKBAEEAUoFAAEBSgQBBgFMBQADAUwEAQYBTAUAAwFMBAEGAUwFAAMB
+gAQBBAFABQABAUAEAQABYgUAAAFIBQAGAVQFAwMBSAQABAF4BQABAXgEAQAAAAFABQMCAUwFAwAA
+BAFQBQABAUAEAQYBXAUDAwGABAAGAVwFAAMBXAQBBgFUBQADAVgEAQYBXAUAAwFcBAEEAUoFAAEB
+SgQBBAFYBQABATwEAQYBZAUAAwFYBAEEATgFAAEBOgQBBAFAAwMBATQDAAQDOAADQAsDOBQDAQIu
+AAA0CQAEATgFAAEBOAQBBAFOBQABAYAEAQQBVAUAAQFMBAFBRQFBUgFBVAFBVQJCRQFCRwFCTgFC
+UgRDQQJDSAFDWQFDWgFERQNESwFFRQFFUwFGSQFGUgFHQgFHUgFIUgFIVQFJRQFJUwFJVAFKUANL
+UgNLVwFMSQFMVAFMVQFMVgFNQQFNVAFNWAFOTAFOTwFQTAFQVAFQWQFSTwFSVQFTRQFTRwRTSQFT
+SwFUUgdUVwJVUwRBRQFBVQJCTgFKUAFLUgFNWAFSVQFVUwEABAAAAAQZAABBVAAASlAHAEtSEgBR
+MhwAUlUeAFNBIwBVUyUAWFQvAAAAPwABAAAAFgIAAAEAAACCAQAABAAAAB0CAAAFAAAAKDYAAERF
+AAADAAAAbQcAAEpQAAAKAAAAuQUAAFVTAAADAAAAaAAAAAoMDQ4PEBESExQVAhQVCAwNDg8QERIT
+CAQFBgcICQoLAAAAAAEBAAQCACIDACMEAD4FA+QGAAAAAAEBAAMCAAUDAC0EADoFAGcGAGgHA6oI
+A8cJA8gKAAAAAAEBAAQCABgDAC0EADAFADEGAEYHAEcIA74JAAAAA+EBAAAAAAUBAA0CA9UDA9wE
+AAAAA+QBAAAAAAEBABkCABoDAC4EA1wFA2YGA3MHA3QIA9wJAAAAAE8BAFACAFEDAFIEAFMFAFQG
+AFUHAFYIAFcJAFkKAFoLAFsMAFwNA9MOA9QPAQ4B4BoAANgFAAAPBgAASzYAAMkGAABIGQAAOQIA
+AIAtAAAMFQAA4AcAADAbAACUBwAA/DUAAH+yAALYGAAAAAAAAIAAAABoNgAAAAAAAAQ2AAD5FAAA
+AAAAAOAFAACcBwAAzAcAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAdAAAAAEAAAAlGgAASUQBAAgA
+AAC/GAAAVVMBAAcAAAB8BwAAWEEAAAIAAAB2BwAAWFQAAC8AAACJAQAAWFYAADAAAADyAAAAWFoL
+ADEAAAAsGAAACAIAADgbAAAjYQApGSsbAAMjbgAqC///8AAjcgApGiscAAMwQQAOC///8BAwQgAA
+EgAYABAwQwAOCSQJABFBQwAqC///8BBBRAAOOiVHADFBRQAOCw0LACBBRQEOOg1RACBBRQYOYSRq
+ADFBRgAOOiVHADFBRwAOXg1nACBBRwIOFCQaADNBSQAMFAwaABNBSQEFEwkZAANBTAAOOg1RACBB
+TAIOOiVHADFBTQAOCCQHAAFBTgAOEg0YACBBTwAOC///8BBBUQAqC///8BBBUgAOIg0jAABBUgEO
+Eg0YAABBUwAMFAwaABNBUwwFEwkZAANBVAAOOg1RACBBVAEOUg1EACBBVAIOOiVHADFBVAMRWx5f
+ACFBVAQqWy9fACFBVAUwYDhkACFBVAYWEB8RACFBVQAuXzdjAAFBVQIuXzZiAABBVQYOXyRjABFB
+VwAOSA04AAFBVwIOFCQaABNBWAAqC///8BBBWgAOEg0YACBBWgIOOiVHADFCQQAOOg1RACBCQQIO
+OiVHADFCQgAOHSQfABFCRAAOZQ11AABCRAEOZR1wAAFCRQAOOg1RACBCRQEOUg1EACBCRQQOOiVH
+ADFCRgAOOiVHABFCRwAOOg1RACBCRwEOUg1EACBCRwQOOiVHADFCSAAOHA0eAABCSAQOHSUfABFC
+SQAOOiVHABFCSgAOOiVHADFCTAAqC///8BBCTQAMFAwaABNCTQwFEwkZAANCTgAOC///8ABCTgEr
+XjBnAABCTgQsXjMfABFCTwAOFCQaACNCUQAqC///8BBCUgAOYQ1uACBCUgIOYR1qACFCUgQOYiRr
+ACFCUwAuHDYeAABCUwIMFAwaABNCVAAOCyQLADBCVgAqC///8BBCVwAOZCRtADFCWQAOOv//8ABC
+WQMOOiVHAAFCWgAOFCQaABNDQQAAHAAeAABDQQIMFAwaABNDQVIHLwQ3AANDQVMqLy83AANDQVUJ
+MgYxAARDQwAqC///8BBDRAAOOiVHADFDRgAOOiVHABFDRwAOOiVHADFDSAAOOg1RACBDSAEOUg1E
+ACBDSAQOOiVHADFDSQAOOiVHADFDSwAOYyRsABFDTAAOCg0KAAFDTQAOOiVHABFDTgAzaDpvAABD
+TiYOHiQgABFDTwAOEQ0PACBDTxEOFCQaADNDUAAqC///8BBDUgAOEg0UACBDUhEOFCQaADNDVQAO
+CyX/wDBDVgAOOiVHABFDVwAqC///8BBDWAAuXzZjABFDWQAOOg1RACBDWQEOUg1EACBDWQQOOiVH
+ADFDWgAOOg1RACBDWgEOUg1EACBDWgQOOiVHADFERQAOOg1RACBERQMOUg1EACBERQcOOiVHADFE
+SgAOC///8BBESwAOOg1RACBESwEOUg1EACBESwQOOiVHADFETQAOFCQaABNETwAOFCQaABNEWgAO
+CA0IACBFMAAODw0SACBFQwAOEg0YACBFQxUOGB0VACNFRQAOOg1RACBFRQEOUg1EACBFRQQOOiVH
+ADFFRwAOXi1oAABFRw0OXiVpABBFSAAOCSQJABFFUgAOC///8BBFUwAOOg1RACBFUwEOUg1EACBF
+UwQOOiVHADFFVAAOOg1RAABFVAIOOiVHABFGSQAOOg1RACBGSQEOUg1EACBGSQQOOiVHADFGSgAO
+XyRjABFGSwAOOiVHABFGTQAMFAwaABNGTwAOOiVHADFGUgAOOg1RACBGUgEOUg1EACBGUgUOOiVH
+ADFHQQAOOiVHABFHQgAOOg1RACBHQgEOUg1EACBHQgYOOiVHADFHRAAOYQ1uAABHRAIOFCQaABNH
+RQAOOiVHADFHRgAOOg1RAABHRgIOOiVHABFHRwAOOiVHABFHSAAOOiVHADFHSQAOOiVHABFHTAAq
+C///8BBHTQAOOiVHABFHTgAOOiVHADFHUAAOOg1RAABHUAIOOiVHABFHUQAOOiVHABFHUgAOOg1R
+ACBHUgEOUg1EACBHUgQOOiVHADFHUwAqC///8BBHVAAOEhwYABBHVAEOGB0VAANHVQAAEgAYAABH
+VR4NMwctAANHVwAOOiVHABFHWQAOZSRwABFISwAuYTduAABISwIvYTdqAAFITQAqC///8BBITgAO
+ZiR3AANIUgAOOg1RACBIUgEOUg1EACBIUgQOOiVHADFIVAAOZSRwAAFIVQAOOg1RACBIVQEOUg1E
+ACBIVQQOOiVHADFJRAAOCw0LAABJRAEOcQ18AABJRAUObyV9ABBJRCYYcCF+AABJRQAOOg1RACBJ
+RQEOUg1EACBJRQUOOiVHADFJTAAOCA0IACBJTA4OSSQ6ACFJTC4XXCBgAAFJTQAOOiVHABFJTgAr
+XjBnACBJTgMrXjRmADFJTwAOOiVHABFJUQAOOiVHABFJUgAOCyX/wDBJUwAOOg1RACBJUwEOUg1E
+ACBJUwQOOiVHADFJVAAOOg1RACBJVAEOUg1EACBJVAQOOiVHADFKMAAOUydQAABKMQAlD///8ABK
+MgAlCP//8ABKMwAlU///8ABKNAAlIf//8ABKNQAlJP//8ABKNgAlJf//8ABKNwAlJv//8ABKOAAl
+Xf//8ABKOQAOJSclAABKRQAOOiVHABFKTQAOZSV2ABFKTwAOAQ0BACBKTwMOASUAADFKUAAlWipN
+AABKUAElJSclAABKUAIlWidPAABKUAMOWg1IAABKUAQoWipOAAFKUAUoWSpOAAFKUAYnVilKAAFK
+UAcqVy9LAAFKUAg0WDtMAAFKUAkmVShJAAFKUAomVChJAAFLQQAyFzkXAARLQwAyZTlwAARLRAAO
+ag10AARLRQAOOg1GACFLRwAOOiVHABFLSAAOYQ1uAABLSAIOYSRqABFLSQAuXzZjABFLSwAyCzkL
+AABLTQAOOiVHABFLTgAOFCQaABNLUAAqC///8BBLUgAqbyxhAABLUgEOJyxhAABLUgIzLTopABFL
+UgMOKhwoAABLUgQ1KDwrAAFLUgUOKBwrAAFLUgYOKxwmAAFLUgc2KT0qAAFLUgg2LD0nAAFLUgka
+RCNTAAFLVwAOCw0LACBLVwEOCA0IACBLVwUOCCUHADFLWAAOZw1yAARLWQAQCxYLAABLWQMMFAwa
+ABNLWgAOUSRCAABMQQAOSA04AAFMQQIOZCRtABFMQgAOZQ11ACBMQgUOYSRqADFMQwAOFCQaABNM
+SQAOOg1RAABMSQEOUg1EAABMSQQOOiVHADFMSwArYTBuAABMSwErYTFqAAFMUgAOYS1sABFMUwAO
+Og1RAABMUwIOOiVHABFMVAAOOg1RACBMVAEOUg1EACBMVAQOOiVHADFMVQAOOg1RACBMVQEOUg1E
+ACBMVQMOOh1GACFMVgAOOg1RACBMVgEOUg1EACBMVgQOOiVHADFMWQAOCyb/wBBNQQAODy0QACBN
+QQEOCC0IACBNQQIOCC4HACFNQwAOOg1RAABNQwEOOiVHABFNRAAOOg1RACBNRAIOOiVHADFNRQAO
+Og1RACBNRQIOOiVHADFNRgAOZCRtABFNRwAOOiVHADFNSAAqC///8BBNSwAOOg1RACBNSwIOOiVH
+ADFNTAAOOiVHABFNTQAOOiVHABFNTgAOEhwYACBNTgEOGB0VACNNTwAOaQ11AABNUAAMFAwaABNN
+UQAOOg1RAABNUQIOOiVHABFNUgAOOg1RAABNUgIOOiVHABFNUwAOOiVHADFNVAAOOg1RACBNVAEO
+Ug1EACBNVAQOOiVHADFNVQAOOg1RACBNVQIOOiVHADFNVgAODQ0OACBNVgMODhwNADFNVwAOZQ11
+ACBNVwEOZCRtADFNWAAOCA0IAABNWAEOHA0eAABNWCwNMwctAANNWQArXjBnAABNWQMrXjJlAAFN
+WgAOFSQTABNOQQAOFSQTABNOQwAOOiVHABFORQAOOiVHABFORgAOXxxiAABORwAOIyUkADFOSQAO
+Eg0YACBOSQIOFCQaADNOTAAOOg1RACBOTAEOUg1EACBOTAQOOiVHADFOTwAOOg1RACBOTwEOUg1E
+ACBOTwQOOiVHADFOUAArDDAMAABOUgAOOiVHABFOVQAOEv//8BBOWgAOYQ1uACBOWgQOYyRsADFP
+TQAOOg1RACBPTQQOOiVHADFQQQArADAiACBQQREOHyQhADNQRQAOEg0YACBQRRQOFCQaADNQRgAO
+OiVHABFQRwAOAg0BAABQSAAOYQ1uACBQSAUOYSRqADFQSwAOaRxxACFQTAAOOg1RACBQTAEOUg1E
+ACBQTAQOOiVHADFQTQAOOiVHABFQTgAqC///8BBQUgAAEgAYAABQUiYNMwctAANQUwAqC///8ABQ
+VAAOOg1RACBQVAEOUg1EACBQVAQOOiVHADFQVwAOFCQaABNQWQAOZQ11AABQWQEOYQ1uAABQWQIO
+YSRqABFRMQACCwILAABRMgAALgAsAABRMgELOAs0AANRQQAOaSVzADFSRQAOOg1RAABSRQIOOiVH
+ABFSTwAOOg1RACBSTwEOUg1EACBSTwQOOiVHADFSUwAOOg1RACBSUwIOOiVHADFSVQAOC///8ABS
+VQEODhwNAAFSVQIOTBw9AABSVQMZTiI/AAFSVQQPTRU+ACFSVwAOOiVHADFTQQAOSA05ACFTQQEt
+SjU7AAFTQgAOC///8BBTQwAOZCRtABFTRAAOCyX/wDBTRQAOOg1RACBTRQEOUg1EACBTRQQOOiVH
+ADFTRwArYzBsACFTRwQrXjFlACFTSAAqC///8BBTSQAOOg1RACBTSQEOUg1EACBTSQQOOiVHADFT
+SgAqC///8BBTSwAOOg1RACBTSwEOUg1EACBTSwQOOiVHADFTTAAOOiVHABFTTQAOOiVHABFTTgAO
+Og1RACBTTwAOC///8BBTUgAOOiVHABFTVAAOOiVHABFTVgAOEhwYACBTVhkOFh0WACNTWAAqC///
+8BBTWQAqC///8BBTWgAOOiVHABFUQQAqC///8BBUQwAOOiVHABFURAAOOiVHABFURgAOOiVHABFU
+RwAOOiVHABFUSAAOYQ1uACBUSAUOYSRqADFUSgAOOiVHABFUSwAqC///8BBUTAAOOiVHADFUTQAO
+OiVHABFUTgAOCA0IACBUTgEOCCUHADFUTwAOC///8BBUUgAOCA0IACBUUgcOOiVHADFUVAAOYQ1u
+ACBUVAMOYRxqADFUVgAOC///8BBUVwAAawB7AABUVwEFbQl5AANUVwIAbAB4AABUWgAOOiVHABFV
+QQAOC///8ABVQQgOTxxAAABVQRAOUCVBADFVRwAOSA04ACFVTQAAEgAYAABVUwALNgsyAANVUwEL
+NQs1AANVUwIiCxoLAABVUwMECwELAABVUwQKNAouAABVUwUJMQYwAARVUwYIMAUvAARVUwcqOS82
+AANVUwgGOQg2AANVUwkLNwszAANVWQAuYTZuAABVWgAOOiVHABFWQQAOOg1RAABWQQIOOiVHABFW
+QwAOFCQaABNWRQAOXg1nAABWRQMrXjRlABFWRwAOOg1RAABWRwIOFCQaABNWSQAAEgAYAABWTgAO
+CA0IAABWTgQOYxxsABFWVQAOYSRqABFXRgAOOiRRABBXUwAOYRxqABFYMAADGwAdAABYMQAOXg1n
+AABYMgAxAw0EAABYMwAOOg1DACBYQQAOA///8ABYQgAAIP//8ABYUgABbgN6AABYUwAjBxsDACFY
+VAAiCxoLACBYVAETOw9XACFYVAIUPBBYAAFYVAMUPRBZAAFYVAQVCxELACBYVAUVPhFaACFYVAYV
+PxFbAAFYVAcVQBFcAAFYVAgVQRFdAAFYVAkUQhBeAAFYVAoVQxFSACFYVAsbRRJUACFYVAwcRhNV
+ACFYVA0dRxRWACFYVA4hBRkGACJYVA8gBBgFACJYVQASUg5FACBYVgAfCxcLACBYVwAySzk8ACNY
+WAAeOv//8CBYWQAOOg1RACBYWgAkC///8CBYWgsjBhsCACFZMQAAEv//8BBZMgAAEv//8BBZMwAA
+Ev//8BBZNAAAEv//8BBZNQAAEv//8BBZNgAAEv//8BBZNwAAEv//8BBZRQAOZCRtABFZVAAOOg1R
+AABZVAIOOiVHABFZWQAqC///8ABaMQAqC///8BBaMgAqC///8BBaMwAqC///8BBaNAAqC///8BBa
+NQAqC///8BBaNgAqC///8BBaNwAqC///8BBaOAAqC///8BBaOQAqC///8BBaQQAOOg1RACBaQQYO
+OiVHADFaTQArXjBnAABaVwAOOiVHADELAv8DERoYLR5YAAQ4GgNGLAM6NQNGWAMUAP8CFxoXWAIC
+VBoDVFgDFAD/AhcaF1gAAkIaA0JYAycBBAQXGhEtFz4eWAICVCADVFgDIAL/AhccHlgAAk4cA1RY
+AygC/wIXIR5YAAROHANUOQNSSgNUWAMoAQQCFyEeWAAEOBoDOi0DPD8DRlgDKAEKAhchHlgABDga
+AzotAzw/A0ZYAxUB/wEXHAIBVBwDFQH/ARccAgFUHAMgAf8CFRwVWAICVBwDVFgDAAD/AAAAHwH/
+AhccF1cAAkIcA0JXAyAB/wIXHBRYAAJCHANAWAMgAf8CFxwUWAICXBwDUFgDCQD/ARcaAgFUGgMm
+Af8BFyAAA04cA1Q5A1JJAwkA/wEUGgABOBoDDwL/AxEaGDAeWAAFOBoDRiwDOjUDRD4DRlgDEAL/
+AxEaGDEeWAAHOBoDTCwDRDUDQDgDTEUDQFIDTFgDEgL/AxEaGDEeWAAFOBoDTiwDQDUDRDgDTkcD
+KgH/AhccHkACAlQhA3BYAxAC/wMRGhgxHlgABjwaA1AtA0w9A0hSA0xWA1BbAxQC/wIRGh5YAAI6
+GgNQWAMSAv8DERoYMR5YAAc6GgNUKwNMNANIOANQRQNEUQNQVQMrAP8BHiQAAXgkAysC/wEeJAAB
+eCQDCwIFAxEaGC0eWAAEOBoDRiwDOjUDRlgDCwL/AxEaGC0eWAAEOBoDQiwDPDUDQlgDIAL/Ahcc
+IVgCAlQcA1RYAyAB/wIXHBtYAgJUHANkWAMLAv8DERoYLR5YAAQ4GgNOLANANQNOWAMLAgQDERoY
+LR5YAAQ4GgNGLAM6NQNGWAMxAv8BHj4CAVQ+Ay0C/wIVLR5YAAI8LQNEWAMtAf8CHi0eWAICcC0D
+cFgDBQIAAhcWFxoCAlQWA1QaAwcCAAIXFhccAgJUFgNUHAMIAgADFxYXHB4+AgJUFgNUIAMGAv8E
+DhYOGhQvFFcABDAWAzAaA0AvA0BXAw4C/wMOGhQvFFcABEQaA1ItA1E7A0hXAw4C/wMOGhQvFFcA
+A0QaA1QvA1RXAw4C/wMOGhQvFFcABTgaA1AsA0g2A1BDA1BXAxMA/wIOGhRXAAI4GgNUVwMTAP8C
+DhoUVwACRBoDVFcDBgH/BA4WDhoULxRXAAQwFgMwGgM8LwM8VwMUAP8CERoeWAACOBoDRFgDIgL/
+AxgeGE8eWAAHRBoDTCwDRDYDPEEDRE4DOlQDRFgDKAL/AhghHlgAB0wXA0omA1Y1A0o9A0hRAz5U
+A1RYAwwC/wQRGhguGE4eWAAFLhoDKCwDIDcDIk4DJFgDDAL/BBEaGC4YTh5YAAcuGQMqKAMoLAMg
+NQMhOgMiTgMkWAMNAv8EERoYLhhPHlgABEAaA1AuA1BPA0xYAwwC/wQRGhguGE4eWAAFOBoDQC0D
+RDoDRE4DRFgDDQL/BBEaGC4YTx5YAAY+GgNaLANQNgNaQQNWTwN2WAMRAv8DERoYMR5ZAAY+GgNa
+LANQNgNaRANWTwN2WQMoAv8DERoYMR5YAAVaGwNQNgNaRANWTwN2WAMiAv8EERoYLhhPHlgABVob
+A1A2A1pBA1ZPA3ZYAygC/wIYIR5YAAdEGgNMLANENgM8QQNESwM4VANAWAMmAf8CFxwePgIBVCAD
+JwIKAxccHj4eWAAIMBcDOCUDMCkDODMDMDYDOEUDMFEDLFgDJgIDAhccHj4AASggAycB/wMXHB4+
+HlgAAiggAyRYAxUCCgEXHAABKBwDMgD/AR5XAAEkVwMkAgoDFxwePB5XAAIoHwMkVwMhAv8EFxwe
+Oh5OHlgAAygeAyhOAyRYAyACAgIXHB5YAAIoHAMkWAMfAv8CFxweVwACKBwDJFcDFwH/AxccHjse
+VwAEThwDVDkDUkgDUlcDFgIJBBccHjoeTh5YAAQgHAMcOgMcTgMcWAMhAQkEFxweOh5OHlgACTAX
+AzglAzApAzgzAzA2AzhBAzhNAzBRAyxYAx0BAQMXHB5OHlcACDAXAzglAzApAzgzAzA1AzhNAzBR
+AyxXAykB/wEXIgIBVCIDKgEHAxccHj8OWAIBVCMDGgH/AxccHj8OVwAEThwDVDkDUkoDVFcDFAAL
+AhEaFFgAAjgaAzxYAwoB/wQRGhgtHk4eWAIEVBoDTC0DTE4DTFgDHAH/AhQcFE4CAlYcA1ZOAx4B
+/wIUHBRQAANOHANSTwNUWAMlAf8CFx8XWAICVB8DVFgDGAH/AxccGDwYWAIDVBwDWDwDWFgDHAH/
+AhQcFE4CAkgcA0hOAyYB/wIXHB4+AgFUIAMmAv8BFCAAAj4cA0I+AyYC/wEUIAACYhwDZj4DJgIJ
+ARQgAAJiHANmPgMmAv8BFCAABEQZA0IqA0A1A0g+AyYC/wEUIAADQhsDQDUDSD4DJgL/ARQgAAFO
+IAMmAgkBFCACAlQcA3A+AxkC/wIUHBw+AgJUHANwPgMnAf8CFyAXWAACRCADJFgDFQH/ARccAAFO
+HAMEAP8BFxYCAVQWAyAB/wIXHB5YAgJUHANUWAMjAf8DFxweOh5QAgJUHgNUUAMmAf8CFxwkPgAC
+QBoDODADKgH/AxccGD8eWAIBVCMDJwL/AxccGD4eWAICVCADVFgDKgH/AxccGD8eWAIBVCMDGwH/
+AhccHkACAlQcA3BAAzMA/wEbWAIBVFgDMwD/AR5YAAFOWAMUAP8CGxobWAICVBoDVFgDMwD/ARtY
+AgFUWAMzAP8BFFgCAUxYAzIA/wEUVwIBTFcDMAL/AhE0HlgAAkQ0A0RYAy4C/wMRNBg+HlgAAzg0
+A0Q+A0RYAy8C/wMRNBg/HlgABDgzA0A1A0Q4A05HAywC/wMRLRg+HlgAAzgtAzo+Az5YAzIA/wEb
+VwIBVFcDMgD/AR5XAAFSVwMyAP8BG1cCAWRXAwAGAAAA6BoAABAEBQYHCAkKCwwNDg8QERITEgQF
+BgcICQoLDA0ODxAREhMUFQAGAgUFAQEHAgQBAQECCgQCAgEBAQ8CAgEBAQEVBAICBQEBARYCGlcB
+LQItPwH/AVwBEgFYARMBGgMtP1gCGlgqKio6KmoqeiqKKpsqqzo6Omo6impqanpqimqbenp6inqb
+equKioqbm5ubq0VDAAA=
